maps/artemis/planets.dmm

```diff
@@ -766,6 +789,12 @@
 	},
 /turf/simulated/floor/black,
 /area/ship/artemis)
+"uf" = (
+/obj/machinery/atmospherics/pipe/vent{
+	dir = 4
+	},
+/turf/simulated/floor/airless/engine/vacuum,
+/area/ship/space_canvas)
```

Comment:
Why there a vent?!?

Reply:
the vent is there to get rid of filtered out gasses

Reply:
Is there a benefit from simulating that exchange instead of simply removing it?

Reply:
No, but I not familiar with any other atmos system that just deletes it.

Reply:
artemis engine has been changed to be unary and now deletes removed gas